src/Pages/Login/DetailPage/DetailPage.js

```diff
@@ -94,11 +119,16 @@ class DetailPage extends Component {
             </div>
             <div className="likeArea">
               <button className="likeCount">
-                <li class="fas fa-heart"></li>
-                <span>{detail.likes_count}</span>
+                <button
+                  onClick={this.handleChangeColor}
+                  style={{ color: this.state.color }}
```

Comment:
여기도! 구조분해할당하면 아래와 같이 작성할 수 있겠네요 
```suggestion
                  style={{ color }}
```